core/server/worker/src/main/java/alluxio/worker/block/management/BlockManagementTaskResult.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.worker.block.management;
+
+import com.google.common.base.MoreObjects;
+import net.jcip.annotations.NotThreadSafe;
+
+import java.util.Hashtable;
+import java.util.Map;
+
+/**
+ * Holds results of a {@link BlockManagementTask}.
+ *
+ * It contains sub-results for each {@link BlockOperationType} that is issued
+ * by the magement task.
+ */
+@NotThreadSafe
+public class BlockManagementTaskResult {
+  /** Map of block-operation results. */
+  private Map<BlockOperationType, BlockOperationResult> mBlockOpResults;
+
+  /**
+   * Creates a new empty task result.
+   */
+  public BlockManagementTaskResult() {
+    mBlockOpResults = new Hashtable<>();
```

Comment:
Any reason we want to use `HashTable` here?

Reply:
no reason. just accidents. fixed.